ivadomed/scripts/automate_training.py

```diff
@@ -220,13 +224,18 @@ def automate_training(config, param, fixed_split, all_combin, n_iterations=1, ru
     pool = mp.Pool(processes=len(initial_config["gpu"]))
 
     results_df = pd.DataFrame()
+    eval_df = pd.DataFrame
     for i in range(n_iterations):
         if not fixed_split:
             # Set seed for iteration
             seed = random.randint(1, 10001)
             for config in config_list:
                 config["random_seed"] = seed
-
+                if all_logs:
+                    if i:
+                        config["log_directory"] = config["log_directory"].replace("_n=" + str(i - 1), "_n=" + str(i))
+                    else:
+                        config["log_directory"] += "_n=" + str(i)
```

Comment:
Does not really matter but I have not much to say.. so I am going to suggest it: `str(i)` --> `str(i).zfill(2)`

Reply:
Updated :D!